### DISCUSSION TITLE
Daikin Altherma Monoblock EDLA06E2V3 Heatpump - X10A

### DISCUSSION THREAD

**Discussion by dandwhelan** [revk/xESP32-Faikout#432] [Q&A]
Hi,

Has anybody ever got a EDLA06E2V3 working on the X10A? I've been having issues with mine using a faikin. A full discussion on the errors have been getting are on the below thread. 

https://github.com/revk/ESP32-Faikin/issues/429#issuecomment-2286021236

If not, do you think it's possible I can flash the faikin with https://raomin.github.io/ESPAltherma/ ?

**Comment by revk** [comment]
There are pads on the back for USB, which you would have to wire to, but if you do, you can reflash anything built for ESP32-S3. It's a 4MB flash + 2MB SPIRAM mondule.

I have a feeling Altherma is the model that is a challenge, as in, we managed to get some messages.

Try ticking `nos21` and unticking `noas` to try *Altherma_S protocol* and see if you get errors. If that works, then it is not a complete implementation, and I don't think there is a way to send controls to the air/con even if it was.

**Comment by dandwhelan** [comment]
Ok, that's good to know. I know it's not possible to control the Heatpump
without a lot of faff. I'm more interested in the data and calculating my
COP.

**Comment by revk** [comment]
OK try that mode, it does, I think, report some stuff and is likely to be extended as we understand more.
